src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -101,4 +100,11 @@ public void deleteSection(Long id, Long stationId) {
         Line line = lineRepository.findById(id).get();
         line.deleteSection(stationId);
     }
+
+    public List<Section> getSectionList() {
```

Comment:
소소한 피드백이지만 메서드 네이밍에 자료구조가 한정되는 것 같아요!

```suggestion
    public List<Section> getSections() {
```